Gems/XR/Code/Include/XR/XRInput.h

```diff
@@ -10,6 +10,7 @@
 
 #include <AzCore/Memory/SystemAllocator.h>
 #include <XR/XRBase.h>
+#include <XR/XRDevice.h>
```

Comment:
I fixed this locally too, but instead of including the header, you can just forward declare `class Device;` below in the namespace.

Reply:
done